src/pages/whitepaper/index.tsx

```diff
@@ -120,6 +106,20 @@ export default function ReductAI(): JSX.Element {
             </div>
           </div>
 
+          {/* Download Block */}
+          <div className={styles.downloadBlock}>
+            <p>
+              Explore ReductStore: A new solution for managing unstructured time
+              series data.
+            </p>
+            <button
+              onClick={openModal}
+              className={"button button--lg button--primary"}
+            >
+              Download White Paper
+            </button>
```

Comment:
please, check numbers. I missed this when I've updated the benchmarks.

Reply:
True, I also updated the white paper with the new numbers.